packages/backend/src/api/new-member-survey/content-types/new-member-survey/schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "kind": "singleType",
+  "collectionName": "new_member_surveys",
+  "info": {
+    "singularName": "new-member-survey",
+    "pluralName": "new-member-surveys",
+    "displayName": "New Member Survey",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": false
+  },
+  "attributes": {
+    "title": {
+      "type": "string",
+      "required": true,
+      "default": "🙋‍♀️ Hello"
+    },
+    "color": {
+      "type": "string",
+      "default": "#5700df",
+      "required": true
+    },
+    "description": {
+      "type": "text",
```

Comment:
If this is intended for discord embeds then we should make this richtext.

Reply:
Aye yeah prob a good idea